### ISSUE TITLE
Looking for a new maintainer

### ISSUE THREAD

**Issue by themasch** [themasch/node-ebml#39]
TL;DR: I'm looking for someone to be the new maintainer for node-ebml. I failed to do this job way to long and finally convinced myself to look out for someone to take over. 

[long-story]
I started this back in 2013 to learn node, "how mkv works" etc. I never really expected that people would actually be using this. For nearly five years I'v more or less forgot about this project. Sometimes just getting remineded by github notifications of issues or pull requests (thanks for all of them!). I felt terrible not beeing able to react properly to those since because I didn't even remembered what this thing does and why. I kind of got around it, merged a lot of PRs, reacting to some of the issues etc. But it took way longer than nessessary and most of these responses never matched my own quality criterias. 

There was a lot of traffic on this package in the last months. Many new contributors (thanks!), new features, a lot of rework and new ideas and problems. This is great, but I can not find the motivation (or time) to deal with that. I do not really care enough about this package to make me do it, I never really used it myself besides that little pet project back in 2013 (I guess…). There have been enough broken promises, and certainly more than enough delayed responses. 

So while it's still hard for me to give up on this, Its certainly the best if someone else takes over. 
[/long-story]

**How will this work?**

My plan for this goes like this: If you want this job, I'd like to ask you to comment on this issue why you would be 'the right one' for this jobs. I don't feel like handing this thing the first dev in sight, especially since there might be some people who are actually using this and I don't want someone to push malware into npm. So please tell us what your "vision" for this project is and what you would do with it. 

I'd also ask the contributors to share their opinion on this (or, of course, "apply for the job") since its actually their code. Theres not a lot of mine left ;). 

After some time and (hopefully) some applications and discussion I'll pick a new maintainer and hand over this repository and the npm package.  

**Whats to be done? Depends!**
Theres the 3.0 release right around the corner but I don't feel like doing that while some of these topics are still open: Theres this number/size thing in #34 and all the bignum-or-not discussion around it. Theres the whole "es6 or browserify or both" and "what style guide" issues that come up now since there are more active maintainers. Performance is a topic to, @pawitp just recently fixed a hugh bug which led to a major performance increase (#37) and who knows how many of these low hanging fruits there are left. I certainly don't.

**Comment by bradisbell** [comment]
@themasch Many thanks for your work on this project and efforts in maintaining to this point.  I hope you're able to find someone to keep it successful into the future!

**Comment by jayands** [comment]
My first reaction to this announcement is that I should *not* be the maintainer, I'll be honest with you. I mean, I've been busy with some stuff that I wanted to show you when it was ready (check out the `@babel` branch, if I have pushed it; I'm not at my computer currently), but I don't know if I could be a lead for a project.

As for vision, however, I DO have a general idea of what I want to see from the project: there is a much more complicated fork of this project called `ts-ebml`. But I mention them because they actually have type definitions for all their complexity and I think that should be one of the things on the list. Flow can strip out the types of the production code, and it can also give type definitions and verify that the content is correct from a functional perspective, that what goes in is what is supposed to go in and that you are expecting the correct thing out. 

I'd also like some of the newer stuff that isn't available because of Node 6 support. Using a tool like Rollup with Babel or even just Babel directly, it would be possible to keep the same level of support for LTS Node and Browserify and still use the latest and greatest features, while keeping the same "no external production dependencies" that seems to be the unwritten rule here.

Of course, I'd be in a much better position to argue the above paragraph if I had all the tests working, but I currently have two failures due to timeouts and not calling the `done` function for the simple tags in the encoder.

Also, code coverage, but I mean, the spec is mostly solved, right? There aren't any glaring features in spec that need to be added, right? So I can't think of anything else to add. Electron support is kind of built in to browser support

**Comment by themasch** [comment]
> …the spec is mostly solved, right? There aren't any glaring features in spec that need to be added, right?

Well, that depends on what you want to do. One could go one and actually parse the blocks payload (like for DateUTC, ContentEncodingScope etc). Currently, we are only decoding the raw structure and packing handing the payload to the user.

**Comment by themasch** [comment]
Okay since @jayands is the only applicant for this position picking the "winner" is pretty easy: congrats Jonathan!
